tests/vk-regression/vk-regression.json

```diff
@@ -168,7 +168,11 @@
   "Bitwise Primitive": {
     "digest": "Bitwise Primitive",
     "methods": {
-      "rot": {
+      "rot32": {
+        "rows": 31,
+        "digest": "3e39e3f7bfdef1c546700c0294407fc2"
+      },
```

Comment:
I can't wait to see how many constraints sha256 takes! if it's mostly using no custom gates we end up with a similar efficiency as those guys who originally implemented it